src/Geometry.cpp

```diff
@@ -82,6 +85,15 @@ double Geometry::getWidth() {
 
 
 /**
+ * @brief Returns the total depth (z extent) of the Geometry in cm.
+ * @return the total depth of the Geometry (cm)
+ */
+double Geometry::getDepth() {
```

Comment:
I can see these height, width and depth routines coming back to haunt us in the future - what about `Geometry::getDeltaX()`, `Geometry::getDeltaY()` and `Geometry::getDeltaZ()`?